packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -0,0 +1,146 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as fs from 'fs'
+import { getLogger, Logger } from '../../shared/logger'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { VueWebview } from '../../webviews/main'
+import { ExtContext } from '../../shared/extensions'
+//import { telemetry } from '../../shared/telemetry/telemetry'
```

Comment:
instead of a comment, maybe you can track this in a checklist or add // TODO: add telemetry here?